backend/code/archipelag/market/models.py

```diff
@@ -8,6 +8,13 @@
 
 from archipelag.ngo.models import NgoUser
 
+POINTS_RULES = dict(
```

Comment:
Nice But I think, `POINT_RULES` would be in new file `market/settings.py`

Reply:
added